xfce-base/exo/exo-4.16.0.ebuild

```diff
@@ -21,9 +21,27 @@ DEPEND="${RDEPEND}"
 BDEPEND="
 	dev-util/gtk-doc-am
 	dev-util/intltool
+	dev-util/xfce4-dev-tools
 	sys-devel/gettext
 	virtual/pkgconfig"
 
+PATCHES=(
+	${FILESDIR}/${PN}-4.16.0-allow-cross.patch
+)
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	local -a myeconfargs
+	tc-is-cross-compiler && myeconfargs+=(
+		ac_cv_have_strftime_extension=$(usex elibc_glibc)
```

Comment:
Make it yes. We can revisit when we actually find a system that doesn't support it.

Reply:
Okay, I just dropped `src_configure` since it defaults to being enabled when cross-compiling.